lisa/sut_orchestrator/azure/common.py

```diff
@@ -2025,12 +2026,19 @@ def save_console_log(
             )
         screenshot_raw_name.unlink()
 
-    log_response = requests.get(diagnostic_data.serial_console_log_blob_uri, timeout=60)
-    if log_response.status_code == 404:
-        log.debug(
-            "The serial console is not generated. "
-            "The reason may be the VM is not started."
+    try:
+        log_response = requests.get(
+            diagnostic_data.serial_console_log_blob_uri, timeout=60
```

Comment:
Does it need to extend the timeout?

Reply:
no need, I didn't see the timeout case happen.